src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx

```diff
@@ -25,10 +28,12 @@ import generalErrorImage1 from './images/general-error-1.jpg';
 import generalErrorImage2 from './images/general-error-2.jpg';
 import generalErrorImage3 from './images/general-error-3.jpg';
```

Comment:
Can these be moved before the styles import?

Reply:
✔️